STATUS: INLINE FAIL
MODIFIED: pitt.search.semanticvectors.vectors.ComplexVectorTest.testComplexOrthogonalize
MODIFIED FILE: /home/lichengpeng/test/automated_script/Lucky-Dhakad/semanticvectors/src/test/java/pitt/search/semanticvectors/vectors/ComplexVectorTest.java
CLEANER: pitt.search.semanticvectors.VectorStoreDeterministicTest.testVectorStoreDeterministic_ComplexVectors
CLEANER FILE: /home/lichengpeng/test/automated_script/Lucky-Dhakad/semanticvectors/src/test/java/pitt/search/semanticvectors/VectorStoreDeterministicTest.java
POLLUTER: pitt.search.semanticvectors.CompareTermsTest.testCompareTermsComplexFlatOrthographic
POLLUTER FILE: /home/lichengpeng/test/automated_script/Lucky-Dhakad/semanticvectors/src/test/java/pitt/search/semanticvectors/CompareTermsTest.java
ORIGINAL CLEANER SIZE: 11
NEW CLEANER SIZE: 2
ELAPSED TIME: 133.4539999961853
==========================
@@ -224,0 +224,4 @@
+ {
+     vecStoreComplex = new VectorStoreDeterministic(FlagConfig.getFlagConfig(new String[] { "-vectortype", "complex", "-dimension", "512", "-seedlength", "512" }));
+     Vector v1 = vecStoreComplex.getVector("testTerm");
+ }
